mcux/mcux-sdk/devices/MCXN947/system_MCXN947_cm33_core1.c

```diff
@@ -85,7 +85,7 @@ __attribute__ ((weak)) void SystemInit (void) {
     extern void *__Vectors;
     SCB->VTOR = (uint32_t) &__Vectors;
 #endif
-#enidf
+#endif
```

Comment:
FYI, this typo is correct in the MCUX source repo at https://github.com/nxp-mcuxpresso/mcux-sdk/blob/021190013e4d7c22b21ad27a73b53179b3fe8afd/devices/MCXN947/system_MCXN947_cm33_core1.c#L89

Reply:
@DerekSnell Hi Can we merge this PR or wait to sync up MCUX source repo?
Thanks.